From 10e6a5aec0403ebfda8aabd1a05e3357f58234a3 Mon Sep 17 00:00:00 2001
From: Carl Schwan <carl@carlschwan.eu>
Date: Wed, 16 Feb 2022 18:18:49 +0000
Subject: [PATCH] Update caption to be compatible with AppStream

And remove hardcoded width and height as it will create troubles in the future when
updating the screnshots.
---
 org.kde.nota.metainfo.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/org.kde.nota.metainfo.xml b/org.kde.nota.metainfo.xml
index 4e630ce..909edea 100644
--- a/org.kde.nota.metainfo.xml
+++ b/org.kde.nota.metainfo.xml
@@ -85,7 +85,7 @@ SPDX-License-Identifier: CC0-1.0
   <url type="donation">https://kde.org/community/donations</url>
   <screenshots>
     <screenshot type="default">
-      <caption>Nota</caption>
+      <caption>Nota, a text editor</caption>
       <caption xml:lang="ar">المحرر</caption>
       <caption xml:lang="ca">Nota</caption>
       <caption xml:lang="ca-valencia">Nota</caption>
@@ -109,7 +109,7 @@ SPDX-License-Identifier: CC0-1.0
       <caption xml:lang="uk">Нота</caption>
       <caption xml:lang="x-test">xxNotaxx</caption>
       <caption xml:lang="zh-CN">Nota</caption>
-      <image type="source" width="2464" height="1929">
+      <image type="source">
         https://cdn.kde.org/screenshots/nota/nota.png
       </image>
     </screenshot>
@@ -136,7 +136,7 @@ SPDX-License-Identifier: CC0-1.0
       <caption xml:lang="uk">Нота, діалогове вікно пошуку</caption>
       <caption xml:lang="x-test">xxNota, find dialogxx</caption>
       <caption xml:lang="zh-CN">Nota 的查找对话框</caption>
-      <image type="source" width="3285" height="2052">
+      <image type="source">
         https://cdn.kde.org/screenshots/nota/nota-tabs-find-dialog.png
       </image>
     </screenshot>
-- 
2.35.1

